src/content/docs/en/core-concepts/endpoints.mdx

```diff
@@ -45,7 +45,7 @@ You can also type your endpoint functions using the `APIRoute` type:
 ```ts
 import type { APIRoute } from 'astro';
 
-export const get: APIRoute = async function get ({params, request}) {
+export const get: APIRoute = ({ params, request }) => {
```

Comment:
Thanks @hvlxh!

This is OK, but we should keep the `async` keyword here otherwise things like the `await fetch()` in the example above wouldn’t work:
```suggestion
export const get: APIRoute = async ({ params, request }) => {
```